history

```diff
@@ -0,0 +1,16 @@
+;; -*- mode: emacs-lisp; coding: utf-8-unix -*-
```

Comment:
This file also does not appear to be related to the commit. Another accidental inclusion?